Makefile

```diff
@@ -161,14 +161,14 @@ openrpc-gen:
 lint-imports:
 	@echo "--> Running imports linter"
 	@for file in `find . -type f -name '*.go'`; \
-		do goimports-reviser -list-diff -set-exit-status -company-prefixes "github.com/celestiaorg"  -project-name "github.com/celestiaorg/"$(PROJECTNAME)"" -output stdout $$file \
+		do goimports-reviser -list-diff -set-exit-status -company-prefixes "github.com/celestiaorg" -project-name "github.com/celestiaorg/celestia-node" -output stdout $$file \
 		 || exit 1;  \
     done;
 .PHONY: lint-imports
 
 ## sort-imports: Sort Go imports.
 sort-imports:
-	@goimports-reviser -company-prefixes "github.com/celestiaorg"  -project-name "github.com/celestiaorg/"$(PROJECTNAME)"" -output stdout .
+	@goimports-reviser -company-prefixes "github.com/celestiaorg" -project-name "github.com/celestiaorg/celestia-node" -output stdout ./...
```

Comment:
On the latest version of goimports-reviser the ./... does not work anymore, so please use the diff from https://github.com/celestiaorg/celestia-node/pull/2595/files, which does exactly the same but loops in bash

Reply:
Jeez, now we have to maintain vendored versions of ci tooling, because it is exactly how it works for me. I guess we can achieve it by providing commands to install all ci dependancies with specified versions. 
It was unclear from your PR that the new version of imports-reviser has changed api.

Reply:
CI seems to be happy about new command

Reply:
Let's make sure our local limiting is happy too

Reply:
Work perfectly for me on both old and new imports-reviser version.

Reply:
The current solution doesn't work for me locally

Reply:
I just ran the command. 
`Failed to fix directory: path is not a directory` - the error I see when `./...` is used

Reply:
![image](https://github.com/celestiaorg/celestia-node/assets/22449364/a4942948-fbf6-4cfa-9443-a7c3453e2d53)

Reply:
Fixed in https://github.com/incu6us/goimports-reviser/releases/tag/v3.4.2, but wasn't yet published to my pkg manager

Reply:
Using `go install` now for the latest version, so this work now for me, so resolved